src/views/SingleEventView.vue

```diff
@@ -329,7 +354,28 @@ export default {
   font-weight: bold;
 }
 
+.announcements-list-container {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  margin-top: 12px;
+  margin-left: 36px;
+  margin-right: 12px;
+  margin-bottom: 12px;
+  width: 30%;
+  height: auto;
+  min-height: 35vh;
+  overflow: hidden;
+}
+.announcement-list {
```

Comment:
When the list overflows because it's too long, the container sizing looks a little funny. Can we adjust this so that the bottom portion of the list doesn't look cut off? Maybe adding something like `flex: 1` and `position: relative` to this class would help?

<img width="422" alt="Screen Shot 2020-08-03 at 10 18 38 PM" src="https://user-images.githubusercontent.com/32231130/89245658-4aedba00-d5d7-11ea-96c5-e67d069ad94a.png">